### PR TITLE
workloadccl: add `fixtures import`, which doesn't require a license

### PR DESCRIPTION
**Pull request by danhhz**
`fixtures make` uses IMPORT to create a dataset in our backup format,
which `fixtures load` then can RESTORE. IMPORT does more work than
RESTORE, but RESTORE requires an enterprise license and IMPORT does not.
Additionally, IMPORT doesn't depend on external data, so `fixtures
import` doesn't have to deal with google storage bandwidth costs (and it
would even work on a cluster firewalled off from the public internet).

Release note: None

### CODE REVIEW

**Comment by danhhz** [comment]
@benesch I've still got to figure out how I want to handle the --db flag (IMPORT apparently doesn't support the `into_db` option), but how do you feel about the UX?

File: pkg/ccl/workloadccl/fixture.go [hunk 1]
```diff
@@ -435,6 +417,57 @@ func MakeFixture(
...
+// ImportFixture works like MakeFixture, but instead of stopping halfway or
+// writing a backup to cloud storage, it finishes ingesting the data.
+func ImportFixture(
+	ctx context.Context,
+	sqlDB *gosql.DB,
+	csvServerURL string,
+	gen workload.Generator,
+	dbName string,
+) error {
+	// WIP use dbName.
```
Comment (benesch) [inline pkg/ccl/workloadccl/fixture.go]: Doesn't `IMPORT database.table` work?

**Review by benesch — APPROVED** [review]
UX LGTM!

**Review by danhhz — COMMENTED** [review]
Okay, this should be ready to go

> *[Reviewable](https://reviewable.io/reviews/cockroachdb/cockroach/31831#-:-LPfky6UDF4zHqAbsupr:b-o11hh1)* status: :shipit: complete! 0 of 0 LGTMs obtained

___
*[pkg/ccl/workloadccl/fixture.go, line 429 at r1](https://reviewable.io/reviews/cockroachdb/cockroach/31831#-LPbl4v52OJnCiGDvvFU-r1-429:-LPfkr3oD5yvgyBf53FW:bnxx8e0) ([raw file](https://github.com/cockroachdb/cockroach/blob/f22dadc3af6316b2c56161155ee3ef3f4a02d4c3/pkg/ccl/workloadccl/fixture.go#L429)):*
<details><summary><i>Previously, benesch (Nikhil Benesch) wrote…</i></summary><blockquote>

Doesn't `IMPORT database.table` work?
</blockquote></details>

It does!

**Review by benesch — APPROVED** [review]

**Comment by danhhz** [comment]
TFTR!

bors r=benesch

**Pull request merged**